Sources/SnapshotTesting/Common/View.swift

```diff
@@ -934,6 +935,15 @@ func prepareView(
   }
   let dispose = add(traits: traits, viewController: viewController, to: window)
 
+  if #available(iOS 15.0, *) {
+      if computeScrollSize, let scrollView = viewController.contentScrollView(for: .top) {
```

Comment:
In some cases this is misleading, for example, `computeScrollSize: true` on the caller site, but without effect when the project's deployment target is less than iOS 15.

Having an alternative for backward compatibility that looks for the top-most or bottom-most(?) `scrollView` would be great.

Reply:
It makes sense. Just to better understand, can you expand a little about what do you think should be the alternative for devices with target < iOS 15 ? Thanks!

Reply:
I'm thinking about something that traverses through the subviews and returns the first (topmost) one which matches the predicate of `view is UIScrollView`.

https://stackoverflow.com/questions/32301336/swift-recursively-cycle-through-all-subviews-to-find-a-specific-class-and-appen
https://stackoverflow.com/questions/32151637/swift-get-all-subviews-of-a-specific-type-and-add-to-an-array/45297466

I'm not saying that these implementations are correct, so please use them with a grain of salt, but the idea is something like this.